src/lib/format/protocol_messages.matter

```diff
@@ -222,6 +222,8 @@ client cluster IMProtocol = 0xFFFF0001 {
     kPathsExhausted = 0xc8;
     kTimedRequestMismatch = 0xc9;
     kFailsafeRequired = 0xca;
+    kInvalidInState = 0xcb;
+    kNoCommandResponse = 0xcc;
```

Comment:
I really wish we could just generate all these things from the StatusCodeList.h data file, but that's obviously a followup.  ;)

Reply:
Or statuscodelist from this file? :smile: 

Generally I think we may want some of our structures updated as well. As we added ref, this file should also contain them for pretty decoding.

Reply:
> Or statuscodelist from this file?

Whichever, as long as we have one source of truth that is easy to edit and have everything Just Work.

Reply:
Tennessee mentioned a similar thing offline. I just created https://github.com/project-chip/connectedhomeip/issues/30927

I don't think I will be able to take this one on in the foreseeable future.